app/Models/UserReport.php

```diff
@@ -53,15 +53,15 @@ class UserReport extends Model
 
     protected $guarded = [];
 
-    public function user(){
+    public function user() {
```

Comment:
__[code_sniffer]__
* Opening brace should be on a new line

<!-- Comment by [SideCI](https://sideci.com) -->